### DISCUSSION TITLE
[Feature Request] Healthcheck/Liveliness Endpoint

### DISCUSSION THREAD

**Discussion by rehanone** [anonaddy/anonaddy#435] [Feature Requests]
Please add a health status or liveliness endpoint that can be monitored without the authentication token.

**Comment by willbrowningme** [comment]
Please can you describe exactly what you mean by this? A health check of what, the service status?

There is already a status page [here](https://status.anonaddy.com/).

**Reply by rehanone** [reply]
What I am looking for is a simple health check endpoint like /health that returns 200 OK to know if my self hosted instance is alive. That check then can be used by docker or Kubernetes to know the service health. This would also allow monitoring of the service via an external service like uptime kuma. The purpose is purely for self hosted instances. I checked the api documents but couldn't find anything that would serve as a service health endpoint.

**Reply by willbrowningme** [reply]
Are you not able to use /login to check if a 200 OK response is returned?

**Comment by rehanone** [comment]
I believe that endpoint requires a auth token. I assumed it would not return http 200 if login fails. I will give it a try though. Thanks again for your help. I would suggest adding a dedicated /health endpoint just for this purpose as an enhancement.
